index.html

```diff
@@ -6,33 +6,162 @@
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+      <div class="cart">
+        <h1>Profesional Art <span>Printing Data</span> </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul>
+          <li>Html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" name="See-Project">See Project</button>
+      </div>
+    </section>
+    <!--the about me section-->
+    <footer id="about">
+      <h1>About me</h1>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button type="button" name="Get My Resume">Get My Resume</button>
+      <div class="line"></div>
+      <ul>
+        <li><h3><img src="Resources/Images/Rectangle55.png" alt="LanguageIcon"><span class="aboutlist">Language</span></h3></li>
+        <li><a href="#">Javascript</a></li>
+        <li><a href="#">Ruby</a></li>
+        <li><a href="#">Html</a></li>
+        <li><a href="#">Css</a></li>
+      </ul>
+      <ul>
+        <li><h3><img src="Resources/Images/Rectangle56.png" alt="Frameworks"><span class="aboutlist">Frameworks</span></h3></li>
+        <li><a href="#">Bootstrap</a></li>
+        <li><a href="#">Ruby on Rails</a></li>
+        <li><a href="#">RSPec</a></li>
+        <li><a href="#">CapyBara</a></li>
+        <li><a href="#">Selenium</a></li>
+      </ul>
+      <ul>
+        <li><h3><img src="Resources/Images/Rectangle57.png" alt="Skills"> <span class="aboutlist">Skills</span></h3></li>
+        <li><a href="#">Codekit</a></li>
+        <li><a href="#">Github</a></li>
+        <li><a href="#">Codepen</a></li>
+        <li><a href="#">Gitlab</a></li>
+        <li><a href="#">Terminal</a></li>
+      </ul>
+    </footer>
+    <section id="contact">
+      <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
- [ ] For best practice, Kindly change the `<h1>` tag here to any of `<h2>` to `<h6>`. It's a good practice to have just one  `<h1>` tag on a webpage for the main title of the page. This will improve the SEO of your page.

Reply:
done